java/src/main/resources/settings.properties

```diff
@@ -0,0 +1,6 @@
+TENANT_ID=1a235385-5d29-40e1-96fd-bc5ec2706361
+CLIENT_ID=
+SECRET=
+QUEUE_NAME=
+SERVICE_BUS_NAMESPACE=elexon-iris.servicebus.windows.net
```

Comment:
In the terminology used elsewhere in the repository, we've used "namespace" to just signify the name of the namespace itself within the Azure tenant (e.g. `elexon-iris` here), and Azure's terminology of "fully qualified namespace" where it has the `.servicebus.windows.net` suffix. Could we stick to that here please?

Reply:
Yes, done